docs/api-references/docs.md

```diff
@@ -59,7 +59,7 @@ string
 <td>
 <code>metadata</code></br>
 <em>
-<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#objectmeta-v1-meta">
+<a href="https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#objectmeta-v1-meta">
```

Comment:
you can update https://github.com/pingcap/tidb-operator/blob/00b4df83e6d01166d6daa9836d0bf0a553bfe538/docs/api-references/config.json#L41

and then run `./hack/update-api-references.sh` to update this file.

Reply:
Done, thanks